pkg/v1/tarball/image.go

```diff
@@ -117,6 +117,22 @@ type Descriptor struct {
 // Manifest represents the manifests of all images as the `manifest.json` file in a `docker save` tarball.
 type Manifest []Descriptor
 
+// LoadManifest load manifest
+func LoadManifest(path string) (Manifest, error) {
```

Comment:
I agree we need this but I don't want the parameter to be a `string`, so that you can use this for in-memory tarballs. Can you make this take an `Opener`?